packages/styles/_utils.scss

```diff
@@ -165,6 +165,12 @@ $sides: (top, right, bottom, left);
     right: var(--TPosRight, 12px);
   }
 }
+.multi-line-truncation {
+  display: -webkit-box;
```

Comment:
Doesn't this completely break if you're not in a WebKit browser?

Reply:
![image](https://user-images.githubusercontent.com/67973710/172709593-a968f746-6479-4968-a5ca-49e26ff36138.png)

Reply:
We don't care about IE